src/components/MultiHopTrade/components/TradeInput/components/Claim/Claim.tsx

```diff
@@ -1,26 +1,81 @@
 import { Card, Stack } from '@chakra-ui/react'
 import type { TxStatus } from '@shapeshiftoss/unchained-client'
-import { useMemo, useState } from 'react'
+import { uuidv4 } from '@walletconnect/utils'
+import { useEffect, useMemo, useState } from 'react'
 import { Navigate, Route, Routes } from 'react-router-dom'
 
 import { SharedTradeInputHeader } from '../../../SharedTradeInput/SharedTradeInputHeader'
 import { ClaimConfirm } from './ClaimConfirm'
 import { ClaimSelect } from './ClaimSelect'
 import { ClaimStatus } from './ClaimStatus'
 import type { ClaimDetails } from './hooks/useArbitrumClaimsByStatus'
+import { useArbitrumClaimsByStatus } from './hooks/useArbitrumClaimsByStatus'
 import { ClaimRoutePaths } from './types'
 
 import { cardstyles } from '@/components/MultiHopTrade/const'
 import { TradeInputTab } from '@/components/MultiHopTrade/types'
+import { actionSlice } from '@/state/slices/actionSlice/actionSlice'
+import {
+  ActionStatus,
+  ActionType,
+  isArbitrumBridgeWithdrawAction,
+} from '@/state/slices/actionSlice/types'
+import { store, useAppDispatch } from '@/state/store'
 
 const cardBorderRadius = { base: '0', md: '2xl' }
 const cardMinHeight = { base: 'calc(100vh - var(--mobile-nav-offset))', md: 'initial' }
 
 export const Claim = ({ onChangeTab }: { onChangeTab: (newTab: TradeInputTab) => void }) => {
+  const dispatch = useAppDispatch()
   const [activeClaim, setActiveClaim] = useState<ClaimDetails | undefined>()
   const [claimTxHash, setClaimTxHash] = useState<string | undefined>()
   const [claimTxStatus, setClaimTxStatus] = useState<TxStatus | undefined>()
 
+  const { claimsByStatus } = useArbitrumClaimsByStatus()
+
+  // Temporary poor man's migration: This tab will be removed in a follow-up
+  useEffect(() => {
+    const allClaims = [...claimsByStatus.Available, ...claimsByStatus.Pending]
```

Comment:
Note - this is an us thing for us to test mostly, but also for users to have their existing in-flight claims be populated in action center, assuming they go to claim tab.

So we don't have completed ones here, as historical ones are not notifications, and the to-be-complete (when claimed) *will* be automagically complete when claimed!